src/Telegram.php

```diff
@@ -1285,4 +1284,56 @@ public function getUpdateFilter(): ?callable
     {
         return $this->update_filter;
     }
+
+    /**
+     * Converts the name of a class into the name of a command.
+     *
+     * @param string $class For example FooBarCommand
+     *
+     * @return string for example foo_bar. In case of errors, returns an empty string
+     */
+    protected function classNameToCommandName(string $class): string
+    {
+        if (!preg_match('/^(.+)Command$/', $class, $matches)) {
+            return '';
```

Comment:
Returning an empty string if no command is found, is not good style. Should return null instead.

Reply:
> Returning an empty string if no command is found, is not good style.

I usually agree, except the cases when an empty string is not a valid value (like this case).
To check if the return value is valid we'd need to check if it's not null OR is an empty string: by avoiding null we simply have to check for empty strings.